charts/portal/values-production.yaml

```diff
@@ -465,6 +472,7 @@ druid:
 
   minio:
     # consider changing to mode... standalone/distributed.
+    # Once Portal is installed, it is not recommented to scale up/down the minio replicas
     replicaCount: 4
```

Comment:
Once Portal is installed, minio can not be scaled up or down.

Reply:
Updated